src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/Connection.php

```diff
@@ -238,11 +246,20 @@ public function send(string $body, array $headers, int $delay = 0): void
             $this->setup();
         }
 
-        $this->call($this->getQueueUrl(), [
+        $messageBody = json_encode(['body' => $body, 'headers' => $headers]);
+
+        $parameters = [
             'Action' => 'SendMessage',
-            'MessageBody' => json_encode(['body' => $body, 'headers' => $headers]),
+            'MessageBody' => $messageBody,
             'DelaySeconds' => $delay,
-        ]);
+        ];
+
+        if ($this->isFifoQueue($this->configuration['queue_name'])) {
+            $parameters['MessageGroupId'] = __METHOD__;
```

Comment:
One must be able to change this. One can have multiple message groups for the same fifo queue.